source/components/molecules/TabNavigator/TabNavigatorItem.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import { Descriptor, EventArg, TabActions, TabNavigationState } from '@react-navigation/native';
+import { TouchableOpacity, LayoutAnimation } from 'react-native';
+import PropTypes from 'prop-types';
+import Text from '../../atoms/Text';
+import { PrimaryColor } from '../../../styles/themeHelpers';
+
+const TabBarItem = styled.View`
+  height: 70px;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin-top: 20px;
+`;
+const Row = styled.View`
+  flex-direction: row;
+  justify-content: space-evenly;
+  margin-bottom: 5px;
+`;
+const LabelText = styled(Text)<{color: PrimaryColor}>`
+  color: ${({theme, color}) => theme.colors.primary[color][0]};
+`;
+  
+const Underline = styled.View<{ color: PrimaryColor; visible: boolean }>`
+  width: 70%;
+  margin-left: 15%;
+  height: 2px;
+  background-color: ${({theme, color, visible}) => visible ? theme.colors.complementary[color][0] : 'transparent'};
+  border-radius: 1px;
+`;
+
+interface Props {
+  route: { key: string; name: string };
+  navigation: { emit: (obj: any) => EventArg<'tabPress', any, any>; dispatch: (obj: any) => void };
+  descriptor: Descriptor<Record<string, object>, string, TabNavigationState, {
+    headerShown?: boolean;
+    title?: string;
+    tabBarIcon: () => React.ReactNode;
+    tabBarIconInactive: () => React.ReactNode;
+}, {}>;
+  state: TabNavigationState;
+  active?: boolean;
+  color: PrimaryColor;
+}
+
+const TabNavigatorItem: React.FC<Props> = ({
+  color,
+  active,
+  route,
+  navigation,
+  descriptor,
+  state,
+}) => (
+  <TouchableOpacity
+    key={route.key}
+    onPress={() => {
+      const event = navigation.emit({
+        type: 'tabPress',
+        target: route.key,
+        canPreventDefault: true,
+      });
+      LayoutAnimation.configureNext(LayoutAnimation.Presets.easeInEaseOut);
+      if (!event.defaultPrevented) {
+        navigation.dispatch({
+          ...TabActions.jumpTo(route.name),
+          target: state.key,
+        });
+      }
+    }}
+    style={{ flex: 1 }}
```

Comment:
Is inline styling ok?

Reply:
No, not really, I will fix this. A leftover from quickly testing some things to get the stying to work.

Reply:
Fixed, moved to styled component.